recipes/icu/all/conanfile.py

```diff
@@ -47,7 +48,7 @@ def _is_mingw(self):
 
     def build_requirements(self):
         if self._the_os == "Windows":
-            self.build_requires("msys2/20161025")
+            self.build_requires("msys2/20190524")
```

Comment:
It seems this new version is not working

Reply:
it might the msys, or something with python... let's check it

Reply:
This was done by the bot. The bot should only change stuff which is safe to do. I'm not yet aware of any breakages because of the msys2 update and it helped in many cases, if this is a breakage, than I will apply a fix to the bot so that it doesn't perform this update anymore

Please let me know

I stopped manually the bot for the latest commit push, but in generally the attempt to overwrite the bot's update would end in a ping-pong push war